indigo/settings.py

```diff
@@ -482,3 +484,23 @@
 # Each item in the list should be a tuple of (Full name, email address). Example:
 # [('Indigo Admin', 'indigoadmin@example.com')]
 ADMINS = []
+
+# Sentry configuration
+SENTRY_DSN = os.environ.get('SENTRY_DSN')
+
+# Sample 50% of requests for performance metrics by default or use the configured
+# environment variable which is a number between 0 (0% of errors sent) and 1 (100% of errors sent).
+SENTRY_SAMPLE_RATE = os.environ.get('SENTRY_SAMPLE_RATE', 0.5)
```

Comment:
i think you need to parse the ENV variable value (which will be a string) into a float